### PR TITLE
 initial commit

### PR DESCRIPTION
**Pull request by milen-ale**
- Add the trial code

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+            <p>
+              <strong>1984</strong> - becomes a distinguished professor at Texas
+              A&M University
+            </p>
+          </li>
+    <!-- <li>
+  <p>
+    <strong>1984</strong> - becomes a distinguished professor at Texas
+    A&M University
+  </p>
```
Comment (milen-ale) [inline index.html:134]: please do not leave unused code as a comment in your codebase, unused code to make your code not clean

File: index.html [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+      integrity="sha384-ka7Sk0Gln4gmtz2MlQnikT1wXgYsOg+OMhuP+IlRH9sENBO0LRn5q+8nbTov4+1p"
+      crossorigin="anonymous"
+    ></script>
+  </head>
+  <body>
+    <div class="first-sec">
+      <h1>Dr. Norman Borlaug</h2>
+      <p>The man who saved a billion lives</p>
+    </div>
+    <div class="image-sec">
```
Comment (milen-ale) [inline index.html:27]: please use more semantic tags like section and nav for readable code

File: index.css [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+.image-sec {
+  background-color: white;
+  margin: 20px;
+  padding: 20px;
+  text-align: center;
+}
+.image-sec img {
+  height: 623px;
+}
+.image-sec p {
```
Comment (milen-ale) [inline index.css:22]: use a layout tool that helps you easily position HTML elements on the page

**Review by milen-ale — COMMENTED** [review]
- please check my contents and make all required changes 
   and submit  for review when you are done🤞:-)

**Review by algerina — APPROVED** [review]
Hi @milen-ale 
You've done an amazing job with the code review so far!

Highlights
:heavy_check_mark:  Nice utilization of GitHub inline comments feature in the review
:heavy_check_mark:  Great job spotting the lack of semantic tags


Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

You can also:

Use emojis and/or gifs in your reviews. They make the code review more human-friendly and fun which makes the person being reviewed feel more open to receiving and accepting the feedback alienrobot
Cheers and Happy coding!clapclapclap

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question with @ so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**